### ISSUE TITLE
can model tetect targets?

### ISSUE THREAD

**Issue by Milesld** [zhen-he/tracking-by-animation#11]
hi,thanks for your model. I have a question, can this model detect targets? And before test, need I  detect targets first?

**Comment by zhen-he** [comment]
Yes, you can disable the temporal connection of the RNN (GRU), and then could do detection. You don't need to specify the detections (or bounding boxes) before test, the model will do this automatically.
